msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java

```diff
@@ -154,6 +157,9 @@
     @SerializedName(POWER_OPT_CHECK_FOR_NETWORK_REQUEST_ENABLED)
     private Boolean powerOptCheckEnabled;
 
+    @SerializedName(HANDLE_TASKS_WITH_NULL_TASKAFFINITY)
+    private Boolean handleNullTaskAffinity;
```

Comment:
Does this matter whether it `Boolean` vs `boolean` and do we care/manage `null` values?

Reply:
I followed the lead of the lines above... assume this is required for the tool we're using to generate Equals and GetHashcode... which for some reason is excaping my recollection.